util/rlp/fuzz/fuzz_targets/untrusted.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+#![cfg(feature = "nightly")]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate rlp;
+
+use rlp::UntrustedRlp;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = UntrustedRlp::new(data);
```

Comment:
`UntrustedRlp` has an `iter()` method yielding an iterator over other `UntrustedRlp` elements within it. You could write a function
```rust
fn iter_recursive(rlp: UntrustedRlp) {
    for x in rlp.iter() {
        iter_recursive(x)
    }
}
```

this would essentially walk the whole RLP structure recursively